### DISCUSSION TITLE
Accessing GitHub Student

### DISCUSSION THREAD

**Discussion by Wtreck** [community/community#70220] [GitHub Education] [answered]
![Screenshot_20231016_125258_Chrome.jpg](https://github.com/community/community/assets/61206987/9ca634b7-91ce-4156-ad59-2ddf5b2d2362)

After a week it still says that my application is being processed, is this normal or do I have to reapply because from what I've read it usually takes 72 hours

**Comment by andrej-kocijan** [comment]
In my experience the amount of time can vary quite a bit, don't think a week is a reason to worry for now. If you do find yourself stuck on pending you can get in contact with the staff via https://support.github.com/request

**Reply by andrej-kocijan** [reply, marked as answer]
Hey @Wtreck please feel free to mark the commend as an answer if it did help you :)
